modules/oe_content_entity/modules/oe_content_entity_contact/src/PermissionCallbacks.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_entity_contact;
+
+use Drupal\oe_content_entity\PermissionCallbacksBase;
+use Drupal\oe_content_entity_contact\Entity\ContactType;
+
+/**
+ * Provides dynamic permissions for Contact entity.
+ */
+class PermissionCallbacks extends PermissionCallbacksBase {
+
+  /**
+   * Returns the entity type id.
+   *
+   * @return string
+   *   The entity type id.
+   */
+  protected function getEntityTypeId(): string {
+    return 'oe_contact';
+  }
+
+  /**
+   * Returns the entity type label.
+   *
+   * @return string
+   *   The entity type label.
+   */
+  protected function getEntityTypeLabel(): string {
+    return 'Contact';
```

Comment:
Entity type label can be given by the entity storage, something like:
```
    $storage = \Drupal::entityTypeManager()->getStorage('oe_contact');
    $label_key = $storage->getEntityType()->getKey('label');
```
We can simply move this method in the base class and use the storage to return the label.